spec/bundler_inject_spec.rb

```diff
@@ -3,6 +3,8 @@
     bundler_inject_root = Pathname.new(__dir__).join("..").expand_path
 
     <<~G
+      #{coverage_prelude}
```

Comment:
First off, I like this approach of loading it in the `Gemfile`.  👍 

But, I think to avoid having to include this in the tests, can we merge this as part of `write_gemfile` so it is not something a developer will see when reading the specs?  It is interesting info to know when understanding everything is tied together in CI, but extraneous information when trying to understand the contents of this particular spec.

Reply:
yeah, that's a good idea